src/renderer/helpers/addressHelper.ts

```diff
@@ -55,3 +56,5 @@ export const removeAddressPrefix = (address: Address): Address => {
   const prefixIndex = address.indexOf(':') + 1
   return address.substr(prefixIndex > 0 ? prefixIndex : 0)
 }
+
+export const getETHChecksumAddress = (address: Address) => ethers.utils.getAddress(address.toLowerCase())
```

Comment:
Why `address.toLowerCase()` and not just `address`?

Pls add comment:
```ts
/**
* Helper to get ETH address as a checksum address
*/
```

[nitpick] Pls keep function names in common (`getEthXYZ` vs `getETHXYZ` / `getEthTokenAddress` vs. `getETHChecksumAddress`)